src/main/java/com/vg/hm/samples/usermanagementsvc/rest/api/v1/UserV1RestService.java

```diff
@@ -31,7 +38,7 @@
     private APIValidator updateUserV1Validator;
 
     @Autowired
-    private UserService userService;
+    public UserService userService;
```

Comment:
any particular reason to set this "public"?

Reply:
no, i changed it because i was having problems with the annotation but i forgot to change it back